src/main/resources/application-local.yaml

```diff
@@ -20,11 +20,26 @@ spring:
             prompt: login
             issuer-uri: https://hmctsdartsb2csbox.b2clogin.com/${AAD_B2C_TENANT_ID_KEY:00000000-0000-0000-0000-000000000000}/v2.0/
             provider: external-azure-ad-provider
+          internal-azure-ad:
```

Comment:
I don't think we should be adding the same properties to local, dev profiles if they are already on the default `application.yaml`. We should only be adding the properties here that we want to override

Reply:
This is a placeholder for internal AD and we will have new config/properties once justice team provides it.